guides/micronaut-oracle-cloud-streaming/chess-game/groovy/src/main/groovy/example/micronaut/chess/GameController.groovy

```diff
@@ -0,0 +1,81 @@
+package example.micronaut.chess
+
+import example.micronaut.chess.dto.GameDTO
+import example.micronaut.chess.dto.GameStateDTO
+import groovy.transform.CompileStatic
+import io.micronaut.http.annotation.Controller
+import io.micronaut.http.annotation.PathVariable
+import io.micronaut.http.annotation.Post
+import io.micronaut.scheduling.TaskExecutors
+import io.micronaut.scheduling.annotation.ExecuteOn
+import io.reactivex.Single
+
+import static io.micronaut.http.MediaType.APPLICATION_FORM_URLENCODED
+import static io.micronaut.http.MediaType.TEXT_PLAIN
+
+/**
+ * Endpoints called by the front end using Ajax.
+ */
+@Controller('/game')
+@ExecuteOn(TaskExecutors.IO) // <1>
+@CompileStatic
+class GameController {
+
+    private final GameReporter gameReporter
+
+    GameController(GameReporter gameReporter) {
+        this.gameReporter = gameReporter
+    }
+
+    /**
+     * Start a new game.
+     *
+     * @param b black's name
+     * @param w white's name
+     * @return the id of the game
+     */
+    @Post(value = '/start', consumes = APPLICATION_FORM_URLENCODED, produces = TEXT_PLAIN) // <2>
+    Single<String> start(String b, String w) { // <3>
```

Comment:
This should return `@SingleResult Publisher<String>` instead of `Single<String>`.

Reply:
Why this is not a module?

Reply:
I've created a discussion outside of the PR to come up with a decision: https://github.com/micronaut-projects/micronaut-guides/discussions/393

Reply:
@burtbeckwith can you update the guide to use `reactor` and add the feature `micronaut-reactor`. Thanks!